Description: Fix -Wunused-result compiler warning.
 In particular, check if file given by -F option is empty.  If it is, print
 a warning message.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2015-01-17

--- a/src/wmfire.c
+++ b/src/wmfire.c
@@ -365,7 +365,8 @@
 	/* First number only. Complex parsing should be done in */
 	/* external program and value saved to monitored file.  */
 
-	fgets(buf, sizeof (buf), fp);
+	if (fgets(buf, sizeof (buf), fp) == NULL)
+		fprintf(stderr, "warning: file '%s' is empty\n", file_name);
 	number = atof(buf);
 	fclose(fp);
 
